Connectors release 25.04 - release note (#4273)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md

## 2025

### April

<Tabs groupId="sync">
<TabItem value="New connectors" label="New connectors">

- [**Exense Step REST API**](../procedures/applications-exense-step-restapi.md) - Initial release of Exense Step REST API.

</TabItem>
<TabItem value="Enhancements" label="Enhancements">

> - [**APC UPS**](../procedures/hardware-ups-apc-snmp.md) - Enhance replace las time output to be humanly readable in **battery-status mode**, from a [The Watch idea](https://thewatch.centreon.com/ideas/hardware-ups-apc-snmp-plugin-warn-crit-output-with-options-warning-replace-lasttime-critical-replace-lasttime-3427).

Author: ```suggestion
- [**APC UPS**](../procedures/hardware-ups-apc-snmp.md) - Enhance replace last time output to be humanly readable in **battery-status mode**, from a [The Watch idea](https://thewatch.centreon.com/ideas/hardware-ups-apc-snmp-plugin-warn-crit-output-with-options-warning-replace-lasttime-critical-replace-lasttime-3427).
```

> - [**APC UPS SNMP**](../procedures/hardware-ups-apc-snmp.md) - Enhance replace last time output to be humanly readable in **battery-status mode**, from a [The Watch idea](https://thewatch.centreon.com/ideas/hardware-ups-apc-snmp-plugin-warn-crit-output-with-options-warning-replace-lasttime-critical-replace-lasttime-3427).

Reviewer: ```suggestion
- [**APC UPS SNMP**](../procedures/hardware-ups-apc-snmp.md) - Enhance **replace-lasttime** output to be humanly readable in **battery-status** mode, from a [The Watch idea](https://thewatch.centreon.com/ideas/hardware-ups-apc-snmp-plugin-warn-crit-output-with-options-warning-replace-lasttime-critical-replace-lasttime-3427).
```

> - [**Colubris SNMP**](../procedures/network-colubris-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**Colubris SNMP**](../procedures/network-colubris-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

- [**F5 BigIP SNMP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Added new **cpu-usage** mode, based on community contribution [PR 4701](https://github.com/centreon/centreon-plugins/pull/4701).

> - [**Fortinet FortiMail SNMP**](../procedures/network-fortinet-fortimail-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**Fortinet FortiMail SNMP**](../procedures/network-fortinet-fortimail-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

> - [**Fortinet FortiSwitch SNMP**](../procedures/network-fortinet-fortiswitch-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**Fortinet FortiSwitch SNMP**](../procedures/network-fortinet-fortiswitch-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

> - [**Fortinet Fortigate Rest API**](../procedures/network-fortinet-fortigate-restapi.md) - Added a new mode **certificates**, from community contribution [PR 5473](https://github.com/centreon/centreon-plugins/pull/5473).

Reviewer: ```suggestion
- [**Fortinet Fortigate Rest API**](../procedures/network-fortinet-fortigate-restapi.md) - Added a new **certificates** mode, from community contribution [PR 5473](https://github.com/centreon/centreon-plugins/pull/5473).
```

- [**Fortinet Fortigate SNMP**](../procedures/network-firewalls-fortinet-fortigate-snmp.md) - Added **switch-usage** mode, **switch discovery rule** and **vdom discovery rule**, from community contribution [PR 5474](https://github.com/centreon/centreon-plugins/pull/5474).

> - [**HP-UX SNMP**](../procedures/operatingsystems-hpux-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**HP-UX SNMP**](../procedures/operatingsystems-hpux-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

> - [**IP-Label Ekara Rest API**](../procedures/applications-monitoring-iplabel-ekara-restapi.md) - Steps are now displayed in the actual steps order instead of alphabetical.

Reviewer: ```suggestion
- [**IP-Label Ekara Rest API**](../procedures/applications-monitoring-iplabel-ekara-restapi.md) - Steps are now displayed in the actual step order instead of alphabetical.
```

- [**JMeter**](../procedures/applications-jmeter.md) - Added `--command-path` option as macro in host template.
- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Enhance uptime date display in **uptime** mode, based on community contribution [PR 4853](https://github.com/centreon/centreon-plugins/pull/4853).
- [**Silverpeak SNMP**](../procedures/network-silverpeak-snmp.md) - Added **interfaces discovery rule**.

> - [**Synology SNMP**](../procedures/hardware-storage-synology-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**Synology SNMP**](../procedures/hardware-storage-synology-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

- [**VMware8 ESX REST API**](../procedures/application-virtualization-vmware8-esx-restapi.md) - Added new **diskio**, **network** and **swap** modes.
- [**Veeam Backup Enterprise Manager Rest API**](../procedures/applications-veeam-vbem-restapi.md) - Added handling of replication jobs to **jobs** mode.

> - [**Viptela SNMP**](../procedures/network-viptela-snmp.md) - Added the option `--check-overload` in **uptime** service template.

Reviewer: ```suggestion
- [**Viptela SNMP**](../procedures/network-viptela-snmp.md) - Added the `--check-overload` option to the **uptime** service template.
```

> - [**Windows Centreon Monitoring Agent**](../procedures/operatingsystems-windows-centreon-monitoring-agent.md) - Added new natives checks **cma-health**, **eventlog** and **services**.

Reviewer: ```suggestion
- [**Windows Centreon Monitoring Agent**](../procedures/operatingsystems-windows-centreon-monitoring-agent.md) - Added new native checks: **cma-health**, **eventlog** and **services**.
```

</TabItem>
<TabItem value="Breaking changes" label="Breaking changes">

> - [**F5 BigIP SNMP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Switched metrics to v2 leading to breaking-change state.

Reviewer: ```suggestion
- [**F5 BigIP SNMP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Switched metrics to v2, leading to a breaking-change state.
```

</TabItem>

> <TabItem value="Fix" label="Fix">

Author: ```suggestion
<TabItem value="Bug fixes" label="Bug fixes">
```

> - [**Cisco Standard SNMP**](../procedures/network-cisco-standard-snmp.md) - Fixed an issue concerning wrong calculations after reboot (sysuptime was not considered) in **configuration** mode. Also `status`metric had been changed in `config-running-ahead`, leading to a breaking-change state.

Reviewer: ```suggestion
- [**Cisco Standard SNMP**](../procedures/network-cisco-standard-snmp.md) - Fixed an issue concerning wrong calculations after reboot (sysuptime was not considered) in **configuration** mode. Also, the `status` metric has been changed in `config-running-ahead`, leading to a breaking-change state.
```

> - [**Cisco Standard SNMP**](../procedures/network-cisco-standard-snmp.md) - Fixed an issue of wrong calculation after reboot (sysuptime was not considered) in **configuration** mode.

Reviewer: ```suggestion
- [**Cisco Standard SNMP**](../procedures/network-cisco-standard-snmp.md) - Fixed an issue concerning wrong calculations after reboot (sysuptime was not considered) in **configuration** mode.
```

> - [**Eclipse Mosquitto MQTT**](../procedures/applications-eclipse-mosquitto-mqtt.md) - Fixed perfdata options in mode numeric-value, from community contribution [PR 5462](https://github.com/centreon/centreon-plugins/pull/5462).

Reviewer: ```suggestion
- [**Eclipse Mosquitto MQTT**](../procedures/applications-eclipse-mosquitto-mqtt.md) - Fixed perfdata options in numeric-value mode, from community contribution [PR 5462](https://github.com/centreon/centreon-plugins/pull/5462).
```

> - [**Eclipse Mosquitto MQTT**](../procedures/applications-eclipse-mosquitto-mqtt.md) - Fixed perfdata options in numeric-value mode, from community contribution [PR 5462](https://github.com/centreon/centreon-plugins/pull/5462).

Author: ```suggestion
- [**Eclipse Mosquitto MQTT**](../procedures/applications-eclipse-mosquitto-mqtt.md) - Fixed perfdata options in **numeric-value** mode, from community contribution [PR 5462](https://github.com/centreon/centreon-plugins/pull/5462).
```

> - [**IBM FlashSystem 900 SNMP**](../procedures/hardware-storage-ibm-fs900-snmp.md) - Fixed wrong default CRITICALSTATUS value in **arrays-status** template.

Reviewer: ```suggestion
- [**IBM FlashSystem 900 SNMP**](../procedures/hardware-storage-ibm-fs900-snmp.md) - Fixed wrong default value for CRITICALSTATUS in **arrays-status** template.
```
